client/MentorPanel/src/pages/index.js

```diff
@@ -2,21 +2,21 @@ import React from "react"
 import { graphql } from "gatsby"
 import { connect } from 'react-redux';
 
-import { setMentorResponse } from '../redux/actions'
+import { onMentorLoaded } from '../redux/actions'
 
-import Layout from "../components/layout"
 import Header from '../components/header'
 import Input from '../components/input'
-import VideoPanel from '../components/video-panel'
+import Layout from "../components/layout"
 import Video from "../components/video"
+import VideoPanel from '../components/video-panel'
 
 class IndexPage extends React.Component {
```

Comment:
any reason this has to be an old-style ES6 class/React.Component instead of a functional component?

If it's only because you're using `componentDidMount`, then let's update this to a functional component and you can put the `componentDidMount` code into a `useEffect` hook:

https://dev.to/trentyang/replace-lifecycle-with-hooks-in-react-3d4n